helper_dart/lib/src/user_token_storage.dart

```diff
@@ -1,17 +1,39 @@
-import 'package:shared_preferences/shared_preferences.dart';
+import 'package:hive/hive.dart';
```

Comment:
is hive builtin or just commonly used?

Reply:
It's [commonly used](https://pub.dev/packages/hive)  (top 1% of most used packages).